Gibe.Umbraco.Blog.Tests/BlogServiceTests.cs

```diff
@@ -1,20 +1,142 @@
 ﻿using System;
+using System.Collections;
 using System.Collections.Generic;
+using System.Linq;
+using Examine;
+using Gibe.DittoServices;
+using Gibe.DittoServices.ModelConverters;
+using Gibe.Pager.Interfaces;
+using Gibe.Pager.Models;
+using Gibe.Pager.Services;
+using Gibe.Umbraco.Blog.Filters;
 using Gibe.Umbraco.Blog.Models;
+using Gibe.UmbracoWrappers;
 using NUnit.Framework;
+using Moq;
+using Our.Umbraco.Ditto;
+using umbraco.cms.presentation.create.controls;
+using Umbraco.Core.Models;
 using Umbraco.Core.Models.Membership;
+using Umbraco.Web.Models.TemplateQuery;
 
 namespace Gibe.Umbraco.Blog.Tests
 {
 	[TestFixture]
 	public class BlogServiceTests
 	{
+		private Mock<IPagerService> _pagerService;
+
+		[SetUp]
+		public void SetUp()
+		{
+			_pagerService = new Mock<IPagerService>();
+		}
+
 		[Test]
 		public void GetPosts_returns_all_posts_in_a_page()
```

Comment:
This doesn't necessarily seem like the most useful test

Reply:
This was already there when I started, I can remove it if that's what we want done, I just thought it might be better to not touch what code that was already there.

Reply:
it's probably doing more harm than good here, yeah, because it makes it look like there's a test case covering that behaviour

Reply:
Okay, Ill remove it then no problem.